### ISSUE TITLE
### <span aria-hidden="true">❌</span> Deploy Preview for *oasearch* failed.

### ISSUE THREAD

**Issue by iftybd** [O-A-Notes/oasearch#4]
### <span aria-hidden="true">❌</span> Deploy Preview for *oasearch* failed.

|  Name | Link |
|:-:|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | f8c8a244f9856d08072ae5ea4d1235e3afd2d80d |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/sites/oasearch/deploys/656bfea5d81cbf00088e0dbb |

_Originally posted by @netlify[bot] in https://github.com/O-A-Notes/oasearch/issues/3#issuecomment-1837350086_
